applications/StructuralMechanicsApplication/tests/test_spring_damper_element.py

```diff
@@ -150,7 +150,7 @@ def _set_up_sdof_system(self, current_model):
 
         #create elements
         e01 = mp.CreateNewElement("NodalConcentratedElement3D1N",1,[1],mp.GetProperties()[1])
-        e02 = mp.CreateNewElement("SpringDamperElement3D2N",2,[1,2],mp.GetProperties()[1])
+        e02 = mp.CreateNewElement("SpringDamperElement3D",2,[1,2],mp.GetProperties()[1])
```

Comment:
```suggestion
        e02 = mp.CreateNewElement("SpringDamperElement3D2N",2,[1,2],mp.GetProperties()[1]) # using old name for backward compatibility
```

lets leave one of them with the old name to ensure backward compatibility